pmm-app/src/pmm-dbaas/components/DBCluster/EditDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.types.ts

```diff
@@ -0,0 +1,11 @@
+export enum DBClusterTopology {
```

Comment:
This file doesn't need to be duplicated. Use the one in the AddDBClusterModal folder instead.